app/views/headerFooterTemplate/BeforeBodyContent.scala.html

```diff
@@ -17,10 +17,18 @@
 @import views.html.templates.helpers.BetaBar
 @import views.html.templates.helpers.Breadcrumb
 
-@this(betaBar: BetaBar, breadcrumb: Breadcrumb)
+@this(betaBar: BetaBar, breadcrumb: Breadcrumb, hmrcLanguageSelect : HmrcLanguageSelect)
 
 @(breadcrumbTaxYear: Option[Int] = None, isAgent: Boolean, isOverviewPage: Boolean)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
 @betaBar()
 
+@if(appConfig.welshToggleEnabled) {
+@hmrcLanguageSelect(LanguageSelect(
+language = utils.LanguageUtil.checkLanguage,
+En -> appConfig.routeToSwitchLanguage("english").toString,
+Cy -> appConfig.routeToSwitchLanguage("cymraeg").toString
+))
+}
+
```

Comment:
This could use some indenting, it's a bit hard to read at the minute

Reply:
done